fre/pp/split_netcdf_script.py

```diff
@@ -217,12 +229,15 @@ def set_coord_encoding(dset, vcoords):
   as expected
   we need the list of all vars (varnames) because that's how you get coords
   for the metadata vars (i.e. nv or bnds for time_bnds)
-  dset: xarray dataset object
-  varname: name (string) of data variable we intend to write to file
-  varnames: list of all variables (string) in the dataset; needed to get
-    names of all coordinate variables since coordinate status is defined
-    only in relation with a variable
-  Note: this code removes _FillValue from coordinates. CF-compliant files do not
+  :param dset: xarray Dataset object to query for info
+  :type dset: xarray Dataset object
+  :param vcoords: list of coordinate variables to write to file
+  :type vcoords: list of strings
+  :return: A dictionary where each key is a coordinate in the xarray Dataset and 
+  each value is a dictionary where the keys are the encoding information from
+  the coordinate variable in the Dataset plus the units (if present)
+  :rtype: dict
+  ..note:: This code removes _FillValue from coordinates. CF-compliant files do not
```

Comment:
And then lines 241 to 245 need to be indented to align with the "T" in "This code..."